pyro/poutine/__init__.py

```diff
@@ -123,6 +124,20 @@ def condition(fn, data):
     return ConditionPoutine(fn, data=data)
 
 
+def infer_config(fn, config_fn):
+    """
+    :param fn: a stochastic function (callable containing pyro primitive calls)
+    :param config_fn: a callable taking a site and returning an infer dict
+
+    Alias for InferConfigPoutine constructor.
```

Comment:
nit: You can use sphinx notation to link to the class here:
```py
"""
Alias for :class:`~pyro.poutine.infer_config_poutine.InferConfigPoutine` constructor.
"""
```
The `~` tells sphinx to hide the `pyro.poutine_infer_config_poutine.` prefix, and the result is a clickable link in our docs.